src/Valet/Models/Variable.cs

```diff
@@ -1,6 +1,6 @@
 namespace Valet.Models;
 
-public readonly struct Variable
+public readonly struct Variable : IEquatable<Variable>
```

Comment:
Is there a compelling reason to make this struct `IEquatable<Variable>`? The complexity of maintaining these overridden methods don't aren't tested or seemingly used anywhere 🤔

Reply:
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1815

Reply:
Understood. Since we're not doing any comparison of the `Variable` struct I'd prefer to disable this warning and remove the `IEquatable<Variable>` interface/methods